src/screens/Gurbani/BottomBar.tsx

```diff
@@ -47,7 +49,12 @@ const BottomBar = () => {
 
   return (
     <SafeAreaView style={styles.root} edges={[ 'bottom', 'left', 'right' ]}>
-      <View style={styles.background} />
+
+      <LinearGradient
+        style={styles.background}
+        locations={[ 0.0, 0.733 ]}
```

Comment:
[performance] [non-blocking] It's probably worth moving this to above the component too, as `gradientLocations` or similar, as the array reference will be recreated on each render!

Reply:
Great idea!